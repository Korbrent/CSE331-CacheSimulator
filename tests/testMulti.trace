s 0x0003E801 1
s 0x0003Ea01 1
s 0x0003E601 1
l 0x0003E801 1
s 0x0003E401 1
s 0x0003E201 1
l 0x0003E001 1
s 0x0003E801 1
l 0x0003E801 1
s 0x0003E801 1
s 0x0003E801 1
l 0x0003E801 1
l 0x0003E801 1
s 0x0003E601 1
s 0x0003E601 1
s 0x0003E602 1
l 0x0003E602 1
s 0x0003E802 1
s 0x0003E402 1
s 0x0003E603 1 
l 0x00001B01 1
l 0x0003E801 1
l 0x0014B101 157
l 0x00000001 1
l 0x0003E801 123
l 0x00000101 165
l 0x0003E801 14
l 0x00000201 167865
s 0x0003E801 1678