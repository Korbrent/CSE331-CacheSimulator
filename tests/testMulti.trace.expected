s 0x0003E801 1 m
s 0x0003Ea01 1 m
s 0x0003E601 1 m
l 0x0003E801 1 h
s 0x0003E401 1 m
s 0x0003E201 1 m
l 0x0003E001 1 m
s 0x0003E801 1 m
l 0x0003E801 1 h
s 0x0003E801 1 h
s 0x0003E801 1 h
l 0x0003E801 1 h
l 0x0003E801 1 h
s 0x0003E601 1 m
s 0x0003E601 1 h
s 0x0003E602 1 m
l 0x0003E602 1 h
s 0x0003E802 1 m
s 0x0003E402 1 m
s 0x0003E603 1 m
l 0x00001B01 1 m
l 0x0003E801 1 m
l 0x0014B101 1 m
l 0x00000001 1 m
l 0x0003E801 1 h
l 0x00000101 1 h
l 0x0003E801 1 h
l 0x00000201 1 m
s 0x0003E801 1 h